src/node/interfaces.cpp

```diff
@@ -169,6 +170,27 @@ class NodeImpl : public Node
     }
     void mapPort(bool use_upnp, bool use_natpmp) override { StartMapPort(use_upnp, use_natpmp); }
     bool getProxy(Network net, Proxy& proxy_info) override { return GetProxy(net, proxy_info); }
+    std::string defaultProxyAddress() override
```

Comment:
I don't believe this should return a string. I think it should be an in/out Proxy&.

Reply:
This is just for display purposes, it doesn't make sense to build the object which also involves `CService` and so on, just to come back with eg `ipv4.proxy.ToStringAddrPort();`, I'd leave it as it is but can be done on a follow-up if more ppl think the change is required.